client/components/QuestionsAnswers/QuestionsAnswers.jsx

```diff
@@ -1,13 +1,65 @@
 import React from 'react';
-import { COLORS } from '../../settings/colors';
 import styled from 'styled-components';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import axios from 'axios';
+import api from '../../api.js';
+import QuestionList from './QuestionList.jsx';
+import SearchBar from './SearchBar.jsx';
+
+
+
+class QuestionsAnswers extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      questionsShow: 2,
+      answersShow: 2,
+      questions: [],
+      answeredQuestionShow: 0,
+      answeredQuestion: []
+    };
+  }
+
+  componentDidUpdate(prevProps, prevState, snapshot) {
```

Comment:
This is fine for now - eventually I think we should just have all the dynamic state live in <App> -- sorry for the back and forth on this, didn't become clear to me that this was the correct approach until after building